docs/codebase.md

```diff
@@ -106,7 +109,7 @@ File Descriptions
 
 Public API Index
 ---------------
-  * Table (FixedDataTableContainer.js)
-  * Cell (FixedDataTableCellDefault.js)
-  * Column (FixedDataTableColumn.js)
-  * ColumnGroup (FixedDataTableColumnGroup.js)
+* Table (FixedDataTableContainer.js)
+* Cell (FixedDataTableCellDefault.js)
+* Column (FixedDataTableColumn.js)
+* ColumnGroup (FixedDataTableColumnGroup.js)
```

Comment:
mention the newly exposed components here

Reply:
Done